source/apiVolontaria/volunteer/models.py

```diff
@@ -281,6 +285,25 @@ def nb_volunteers_standby(self):
     def duration(self):
         return self.end_date - self.start_date
 
+    def generate_ics(self):
+
+        cal = Calendar()
+
+        cal.add('prodid', '-//{0}//'.format(self.task_type.name, ))
+        cal.add('version', '2.0')
+
+        ical_event = ICalEvent()
+        ical_event.add('summary', u'NousRire - %s' % self.task_type.name)
+        ical_event.add('dtstart', self.start_date)
+        ical_event.add('dtend', self.end_date)
+        ical_event.add('dtstamp', self.end_date)
+        ical_event.add('location', self.cell.address)
+        ical_event['uid'] = str(self.id)
+
+        cal.add_component(ical_event)
+
+        return cal.to_ical()
```

Comment:
It can be good to move the creation of a calendar file in a generic function so the model just call the function and we doesn't have a big coupling between the calendar lib and our project.

Maybe a generic `functions.py` file ?